lib/git_reflow/commands/refresh.rb

```diff
@@ -0,0 +1,26 @@
+desc 'refresh will rebase your current branch based on the base branch'
+long_desc <<LONGTIME
+  Performs the following:\n
+  \t$ git checkout <base_branch>\n
+  \t$ git pull --rebase <remote_location> <base_branch>\n
+  \t$ git checkout <current_branch>\n
+  \t$ git rebase <base_branch>\n
+LONGTIME
+arg_name '[base_branch] - branch that you want to rebase on (master by default), [remote_location] - remote repository name to fetch updates from (origin by default)'
+command :refresh do |c|
+  c.action do |global_options, options, args|
+    if args.empty?
+      raise "usage: git reflow refresh [base_branch_name - defaults to master] [remote_location - defaults to origin]"
+    else
+      # defaults remote_location to origin
+      remote_location = args.length < 2 ? "origin" : args[1]
+      base_branch = args.length < 1 ? "master" : args[0]
+      current_branch = GitReflow.current_branch
+
+      `git checkout #{base_branch}`
+      `git pull --rebase #{remote_location} #{base_branch}`
```

Comment:
I don't think we want the `--rebase` option here as it may require a force-push to the base branch in the future (which is not something that should ever be done on `master`).

Also, we've recently added `update_current_branch` in [GitHelpers](https://github.com/reenhanced/gitreflow/blob/master/lib/git_reflow/git_helpers.rb#L34).  Go ahead and add the extra logic to first `update_destination(base_branch)` and then perform the extra `git rebase #{base_branch}` you've added here to that method and call that from here.

As a side note, please remember to run any commands with `run` so we can avoid system calls in tests :wink: